src/usr/local/www/diag_arp.php

```diff
@@ -30,10 +30,8 @@
 ##|*MATCH=diag_arp.php*
 ##|-PRIV
 
-@ini_set('zlib.output_compression', 0);
```

Comment:
These removals are intentional.
They were added 8 years ago to output a "loading" dialog to make the page load seem speedy. Unfortunately this messes up with the CSRF magic output handler which was added years later.

Comments near this "loading" dialog (line 278) mention that on modern hardware this should not be a problem. 8 years later I don't think this is a concern anymore.

If page load time is still a concern here, a proper solution can be implemented like loading the actual ARP table via an async AJAX call, not by playing around with output handlers.